docs/skinning.md

```diff
@@ -0,0 +1,116 @@
+# Skinning Your uPortal-home #
+
+_Prerequisites:_
+
+_A working uPortal-app-framework application._
+
+_A code editor._
+
+For a more in-depth discussion of theming, see https://github.com/UW-Madison-DoIT/uw-frame/blob/master/docs/theming.md
+
+1. Locate and open the frame-config.js file in your uw-frame-components/js/ directory.
+
+
+2. Increment the themeVersion in line 9. 
+
+3.  Add the following json to the themes array:
+
+
+```json
+{
+  'name': '',
```

Comment:
yup.  Since this is an example from a .js file, line 19 should use js highlighting